spec/forms/otp_delivery_selection_form_spec.rb

```diff
@@ -42,5 +42,32 @@
         expect(result).to eq result_hash
       end
     end
+
+    context 'when otp_method is the same as the user otp_delivery_preference' do
+      it 'does not update the user' do
+        user = build_stubbed(:user, otp_delivery_preference: 'sms')
```

Comment:
I've noticed while reading this that I am getting confused about where we use `otp_delivery_preference` and where we use `otp_method` and where we use `delivery_method` -- would it be possible to standardize on one? Or at least 2? 

`otp_delivery_preference` makes sense for database, wondering if we can use it for params as well...